docs/atomic-diff.1.md

```diff
@@ -63,6 +69,10 @@ Compare the files and RPMs (without versions) in images 'foo1' and 'foo2' and ou
 Compare only the metadata between images 'foo1' and 'foo2'
 
     atomic diff -m foo1 foo2
+    
+Compare files by 'sha256digests' and 'time' between images 'foo1' and 'foo2'
+
+    atomic diff foo1 foo2 --keywords sha256digest time
```

Comment:
Why not do the "," separated list?  

-k time -k stime -k ppid

Versus

-k time,ktime,ppid

But I guess it is not that big a deal

Most people will just do -k all, or is this the default?